docs/misc/message_system.md

```diff
@@ -108,7 +108,7 @@ Structure of config, types and optionality of specific keys can be found in the
                     },
                     /*
                     For os, environment, browser and transport.
-                    - Values can be array, string or null.
+                    - Values can be array, string or null (except for revision in environment)
```

Comment:
What do you mean by this commit? I mean it is still a string nope?

Reply:
9cb4bc1f9d4db8d8b5eb71bf9817efebabec70c8